src/@next/Tooltip/Tooltip.tsx

```diff
@@ -154,6 +157,9 @@ export const Tooltip = ({
   content,
   preferredPosition = defaultPosition,
   zIndex,
+  clickable = false,
+  timeout = 1000,
+  onClick,
```

Comment:
i think we can just use this to know whether it's clickable or not.

Reply:
you mean use `onClick` to determine whether it is clickable or not?
if that's the case, not sure if it's possible, I tried and it didn't work, the truth value (e.g., `!onClick`) will be false regardless I pass a function or not...

Maybe isn't that the reason why in other components you need something similar like this?
![image](https://github.com/glints-dev/glints-aries/assets/67429929/ec934743-9ba0-45eb-83cc-9d5cb598e3fe)
(there is separate `onBack` and `showBackButton` props here, it could've been merged into one, but I think the reason it's not is because it does not work, unless I'm passing `ComponentAction` that it worked)

also, if we determine using that, it's harder to test and play around in glints-aries (you can't just toggle the `onClick` function in storybook unlike boolean values... lmk if you have alternatives, thanks!